### PR TITLE
[stable/traefik] Added ability to provide extra certs to traefik at deployment time

### PR DESCRIPTION
**Pull request by wf-ryanorth**
Signed-off-by: Ryan Orth <ryanorth@workfront.com>

#### What this PR does / why we need it:
This allows users of traefik to provide as many certificates up-front as they would like. While certs can be provided to each ingress object, having them in a central location is ideal for some situations.

#### Which issue this PR fixes
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [ x] Variables are documented in the README.md
- [x ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by wf-ryanorth** [comment]
hey @dtomcej - could you take a look at this when you get the chance? Thanks!

File: stable/traefik/values.yaml [hunk 1]
```diff
@@ -145,6 +145,14 @@ ssl:
...
     # basic:
     #   testuser: $apr1$JXRA7j2s$LpVns9vsme8FHN0r.aSt11
+  # a list of any extra certificate/key filenames you want included in the traefik instance. You can use this in conjunction with
+  # the "secretFiles" parameter to include the certs on each traefik pod. the expected format is:
+  # extraCerts:
+  #   - certFile: /secrets/cert1.crt
+  #     keyFile: /secrets/key1.key
+  #   - certFile: /secrets/cert2.crt
+  #     keyFile: /secrets/key2.key
+  extraCerts: []
```
Comment (dtomcej) [inline stable/traefik/values.yaml]: Should be commented out by default please.

File: stable/traefik/README.md [hunk 2]
```diff
@@ -135,6 +135,7 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `ssl.defaultIPList`                    | Specify generated self sign cert IP list                                                                                     | `[]`                                              |
 | `ssl.defaultCert`                      | Base64 encoded default certificate                                                                                           | A self-signed certificate                         |
 | `ssl.defaultKey`                       | Base64 encoded private key for the certificate above                                                                         | The private key for the certificate above         |
+| `ssl.extraCerts`                       | A list of any extra certificate/key filenames you want included in the traefik instance. You can use this in conjunction with the `secretFiles` parameter to include the certs on each traefik pod                                                                        | []         |
```
Comment (dtomcej) [inline stable/traefik/README.md]:
```suggestion
| `ssl.extraCerts`                       | A list of any extra certificate/key filenames you want added to the https entrypoint. This must be used in conjunction with the `secretFiles` parameter to include the certs on each traefik pod                                                                        | []         |
```

**Review by dtomcej — CHANGES_REQUESTED** [review]

File: stable/traefik/README.md (hunk 2 above)
Comment (wf-ryanorth) [reply stable/traefik/README.md]: done!

File: stable/traefik/values.yaml (hunk 1 above)
Comment (wf-ryanorth) [reply stable/traefik/values.yaml]: fixed

**Comment by wf-ryanorth** [comment]
thanks for the feedback @dtomcej ! I've updated my changes based on your review. Please let me know if there is anything else you see that needs to be done!

**Comment by ryaneorth** [comment]
hey @dtomcej - could you take another look at these changes? Thanks!

**Comment by ryaneorth** [comment]
@dtomcej - just a gentle poke asking if you could take another look at this PR... If there is someone else I should tag for review, let me know. Thanks!

**Comment by ryaneorth** [comment]
@emilevauge
@timoreimann
@vdemeester
@nmengin
@mmatur
@ldez
@juliens
@jbdoumenjou
@geraldcroes
@SantoDE
@errm

Would one of you mind taking a look at this? Thanks!

**Comment by dtomcej** [comment]
@wf-ryanorth Sorry for the delay. Reading week!

**Comment by ryaneorth** [comment]
Thanks @dtomcej !

**Pull request merged**